logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/EventMapper.kt

```diff
@@ -131,7 +133,11 @@ class EventMapper(
     ) = when (featureConfigUpdatedDTO.data) {
         is FeatureConfigData.FileSharing -> Event.FeatureConfig.FileSharingUpdated(
             id,
-            (featureConfigUpdatedDTO.data as FeatureConfigData.FileSharing).status.name
+            featureConfigMapper.fromFeatureConfigsDTO(featureConfigUpdatedDTO.data as FeatureConfigData.FileSharing)
```

Comment:
I think like other mappers, we can simply name the inner function as 'fromDTO'.

Reply:
👍  I was just following naming that was already used in the class.